themes/bootstrap5/templates/ratings/userlist.phtml

```diff
@@ -0,0 +1,103 @@
+      <?php if ($this->accountCapabilities()->isRatingRemovalAllowed()): ?>
+        <input type="hidden" value="<?=$this->escapeHtmlAttr($this->auth()->getManager()->getCsrfHash())?>" name="csrf">
+        <label>
+          <div class="checkbox">
+            <input type="checkbox" name="selectAll" class="checkbox-select-all">
+            <?=$this->transEsc('select_all_on_page')?>
+          </div>
+        </label>
+        <?=
+          $this->component(
+              'confirm-button',
+              [
+                  'buttonId' => 'cancelSelected',
+                  'buttonName' => 'cancelSelected',
+                  'buttonLabel' => 'delete_selected',
+                  'header' => 'confirm_userrating_delete_selected_text',
+                  'confirmId' => 'confirm_cancel_selected_yes',
+                  'cancelClass' => 'confirm_cancel_no',
+                  'ignoreLightbox' => true,
+              ]
+          )
+        ?>
+      <?php endif; ?>
+      <table class="usercontent-table table-striped table table-responsive">
+        <thead>
+          <tr>
+            <th></th>
+            <th><?=$this->transEsc('Created')?></th>
+            <th><?=$this->transEsc('Title')?></th>
+            <th><?=$this->transEsc('Rating')?></th>
+          </tr>
+        </thead>
+        <tbody>
+          <?php foreach ($this->ratings as $rating):?>
+            <tr>
+              <td class="checkbox-col">
+                <?php if ($this->accountCapabilities()->isRatingRemovalAllowed()):?>
+                  <div class="checkbox-container result">
+                    <div class="checkbox">
+                      <label class="record-checkbox">
+                        <span class="sr-only"><?=$this->transEsc('Select');?></span>
+                        <input class="checkbox-select-item" type="checkbox" name="deleteSelectedrating[]" value="<?=$this->escapeHtmlAttr(($rating['id']))?>" id="checkbox_rating_<?=$rating['id']?>">
+                      </label>
+                    </div>
+                  </div>
+                <?php endif; ?>
+              </td>
+              <td data-label="<?=$this->transEscAttr('Created')?>"><?=$this->dateTime()->convertToDisplayDateAndTime('Y-m-d H:i:s', $rating['created'])?></td>
+              <td data-label="<?=$this->transEscAttr('Title')?>">
+                <a href="<?=$this->url('record', ['id' => $rating['record_id']])?>"><?=$this->escapeHtml($rating['recordTitle'])?></a>
+              </td>
+              <td class="user-rating" data-label="<?=$this->transEscAttr('Rating')?>">
+                <a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getActionUrl($rating['source'] . '|' . $rating['record_id'], 'Rating'))?>" data-lightbox>
+                  <?=$this->render('Helpers/star-rating.phtml', ['readonly' => true, 'ratingData' => ['rating' => $rating['rating'], 'count' => 1]])?>
```

Comment:
When you merge the latest dev code into your branch, you'll need to change this to:

```suggestion
                  <?=$this->component('star-rating', ['readonly' => true, 'ratingData' => ['rating' => $rating['rating'], 'count' => 1]])?>
```

...because of #4317 being completed.

Reply:
Done